company-profiles/ocient.md

```diff
@@ -0,0 +1,31 @@
+# Ocient
+
+## Company blurb
+
+The Ocient Hyperscale Data Warehouse transforms and loads data in seconds, enables organizations to store and analyze more data, and executes queries on hyperscale datasets up to 50x faster.
+
+## Company size
+
+100-200
+
+## Remote status
+
+Remote first company
+
+## Region
+
+* North America
+* Europe
+* India 
+
+## Company technologies
+
+C++, Python, SQL 
```

Comment:
Worth adding Bazel and Java?

Reply:
I have added java and bazel. I hope that's okay @KieranRobson